src/Native/Runtime/thread.h

```diff
@@ -135,7 +135,7 @@ class Thread : private ThreadBuffer
     void ClearState(ThreadStateFlags flags);
     bool IsStateSet(ThreadStateFlags flags);
 
-    static UInt32_BOOL HijackCallback(HANDLE hThread, PAL_LIMITED_CONTEXT* pThreadContext, void* pCallbackContext);
+    static UInt32_BOOL REDHAWK_PALAPI HijackCallback(HANDLE hThread, PAL_LIMITED_CONTEXT* pThreadContext, void* pCallbackContext);
```

Comment:
It would be better to remove `__stdcall` from the other places related to this so that this one is just using the default.